nx-X11/include/Imakefile

```diff
@@ -16,11 +16,8 @@ GLXDIR = GL
 
 SUBDIRS = extensions $(GLXDIR)
 
-AU_HEADERS = Xauth.h
```

Comment:
Well, I have not blindly removed files, I have checked every single file and there was no difference that could harm. I just was not aware that Xauth comes from Xlib now.